### DISCUSSION TITLE
O CNPJ da APyB se enquadra na Lei Rouanet?

### DISCUSSION THREAD

**Discussion by talitarossari** [apyb/comunidade#106] [Geral]
Um possível patrocinador da PySul perguntou se o CNPJ da APyB se enquadra na Lei Rouanet.

**Comment by LanusseMorais** [comment]
Atenta 👀

**Comment by rougeth** [comment]
@talitarossari não acho que seja questão do CNPJ em si, mas sim o tipo de evento que a nossa comunidade organiza. Pelo que eu pesquisei, o foco da lei são iniciativas que fomentam produções culturais e artísticas.

A @apyb/diretoria pode confirmar com alguém saiba mais do assunto, mas pelo que eu entendi, não tem nada que indique que os nossos eventos poderiam ser beneficiados.

No gov.br tem um artigo que cita as principais categorias que podem ser contempladas pela lei:

> Quais formas de expressão são contempladas pela Lei Rouanet?
> ...
> I - artes cênicas - circo, dança, mímica, ópera, teatro e congêneres;
> II - artes visuais - artes gráficas e artes digitais, incluídos pintura, gravura, desenho, escultura, fotografia, arquitetura, grafite e congêneres;
> III - audiovisual - produção cinematográfica e videográfica, rádio, televisão, difusão e formação audiovisual, jogos eletrônicos e congêneres;
> IV - humanidades - literatura, filologia, história, obras de referência e obras afins;
> V - música - música popular, instrumental e erudita e canto coral; e
> VI - patrimônio cultural - patrimônio histórico material e imaterial, patrimônio arquitetônico, patrimônio arqueológico, bibliotecas, museus, arquivos e outros acervos.

https://www.gov.br/secom/pt-br/fatos/brasil-contra-fake/noticias/2023/3/o-que-voce-precisa-saber-sobre-a-lei-rouanet

**Reply by LanusseMorais** [reply]
@rougeth Conforme descrito no Capítulo 2, Seção 1, Art. 4º parágrafo 2º [Instrução Normativa nº1/2023](https://www.in.gov.br/en/web/dou/-/instrucao-normativa-minc-n-1-de-10-de-abril-de-2023-476028057)

> A pessoa jurídica deverá possuir natureza cultural comprovada por meio da existência de código de Classificação Nacional de Atividades Econômicas (CNAE) nos registros do Cadastro Nacional da Pessoa Jurídica (CNPJ), conforme Anexo VII.

Dessa forma para escrever um projeto usando CPNJ é preciso ter um CNAE que comprove a natureza cultural, consegue verificar se a APYB tem o CNAE necessário?

**Reply by naanadr** [reply]
Olá @LanusseMorais , o CNPJ da APyB é o `09.344.485/0001-05` e pesquisando no site da [RF](https://solucoes.receita.fazenda.gov.br/servicos/cnpjreva/Cnpjreva_Solicitacao.asp), temos os seguintes CNAES:
![image](https://github.com/apyb/comunidade/assets/21251472/f074f7bc-7468-4acb-ae95-8f5dc6f35aed)

Vi rapidamente na instrução normativa que você falou, e o CNAE `9493-6/00` está presente no ANEXO VII. Mas não li com calma o conteúdo de toda a norma 😞

**Comment by talitarossari** [comment]
Opa galera @LanusseMorais , @naanadr e @rougeth.
Creio que esse assunto é muito interessante e abre portas para alguns patrocinios interessantes para a comunidade. Que tal marcarmos uma reunião para conversarmos sobre?

**Comment by cecivieira** [comment]
@talitarossari acredito que seria um exercício interessante vocês lerem a Lei em si e se apropriarem mais do assunto.
Cada organização tem a responsabilidade por cumprir todas as exigências das patrocinadoras e essa é uma questão crítica todos os anos. Por exemplo, é muito comum as organizações não prestarem contas à PSF na época devida e isso só ser realizado quando trava a organização do próximo ano.

Entendo demais que no pós evento a organização está exausta, por conta disso o ritmo de trabalho muda ou fica parado. Por isso, peço que vocês se apropriem da temática da legislação e verifiquem os direitos e deveres.

Aproveito para convidar a seguir conversando sobre o assunto aqui na thread. Quais as outras dúvidas e comentários? Acredito que podemos amadurecer a discussão por aqui antes de um encontro síncrono.

**Comment by talitarossari** [comment]
opa @cecivieira
entendo sobre todos os pontos, mas acredito que se estivermos dentro da Lei Rouanet, isso ajudaria inúmeras organizações e abriria portas de grandes empresas de tecnologia que estariam dispostas a patrocinar a PyBR em conjunto.
Não temos nenhum conhecimento juridico, e acredito que apenas ler não vai nos ajudar.
Estamos recorrendo a ApyB justamente pois, acredito eu, que é o grupo/empresa que mais teria meios para validar essa informações, talvez até contratar um auxilio juridico.
Além do mais, a organização da PySul está sem historico e não há saldo devedor de nenhum patrocinador antigo e queremos manter assim. Inclusive abrimos essa discussão para sanar a duvida de um patrocinador que poderia estar patrocinando mais eventos atraves dessa forma.

Porém entendo todos os pontos, obrigada pelo auxilio até aqui.

**Comment by cecivieira** [comment]
@talitarossari Consultamos o escritório de contabilidade para verificar as possibilidades e implicações. Assim que tivermos resposta, seguiremos a conversa nesta thread.

**Comment by PySulBR** [comment]
opa pessoal (@cecivieira), tudo certo?
tivemos algum retorno sobre esse tema dos nossos escritórios os de contabilidade/jurídico?
Encontramos mais alguns patrocinadores nessa modalidade, e que, inclusive, aceitaram a possibilidade de patrocinar a PyBR em conjunto.
Seria muito interessante ao menos ter a certeza que não temos acesso a essa tipo de incentivo.

Abraços,
Talita Rossari, co-organizadora da Python Sul 2024.

**Reply by cecivieira** [reply]
Não é possível termos acesso a esse tipo de incentivo.

**Reply by PySulBR** [reply]
Perfeito! muito obrigada <3
